resources/lang/es.json

```diff
@@ -41,5 +41,6 @@
     "To upgrade, follow the instructions in the docs or use :link-laravelshifts to upgrade automatically.": "Para actualizar, sigue las instrucciones en la documentación o utiliza :link-laravelshifts para actualizar automáticamente.",
     "Update <em>at least</em> to a security-maintained version <strong>as soon as possible!</strong>": "¡Actualice <em>al menos</em> a una versión con mantenimiento de seguridad <strong>lo más pronto que pueda!</strong>",
     "Update to the latest major or LTS release.": "Actualice a la versión principal más reciente o la versión LTS.",
-    "Version": "Versión"
+    "Version": "Versión",
+    "You can also find a list of all of the security advisories for Laravel here: :link": "You can also find a list of all of the security advisories for Laravel here: :link"
```

Comment:
"You can also find a list of all of the security advisories for Laravel here: :link": "También puedes encontrar una lista de todos los avisos de seguridad para Laravel aquí: :link"

Reply:
Commit [3044220](https://github.com/wilsenhc/laravelversions/commit/3044220957ee01619d6292f5a168e633f95ce267) in my fork already adds  this change

Reply:
Thanks, I also put your change into my PR, but you want it too can throw your PR ;)